packages/cli/src/commands/deploy/service.ts

```diff
@@ -43,15 +45,94 @@ export default class Service extends Command {
       },
       {
         title: 'Creating service',
-        skip: () => 'not yet implemented',
-        task: async () => {
-          // service = await create(lcd, definition, credential.mnemonic)
+        task: async (ctx, task) => {
+          const title = task.title
+          const logs = await this.deploy('service', definition, user.uid)
+          logs.on('data', x => {
+            task.title = `${title} (${x.message})`
+          })
+          service = await this.toPromise(logs)
         }
-      },
+      }
     ])
     await tasks.run()
 
-    // this.log(`Service deployed with the hash ${service.hash}`)
-    // this.log(`https://explorer.testnet.mesg.com/services/${service.hash}`)
+    this.log('')
+    this.log(`Service deployed with the hash ${service.hash}`)
+    this.log('')
+
+    const { deploy } = await prompt({ type: 'confirm', name: 'deploy', message: 'Would you like to start this service?' })
+
+    if (deploy) {
+      const env = await this.getEnv(definition.configuration.env || [])
+
+      const logs = await this.deploy('runner', {
+        serviceHash: service.hash,
+        env
+      }, user.uid)
+      logs.on('data', this.log)
```

Comment:
The logging should be improved. Currently it something like:
```
{ data: {}, level: 'info', message: 'runner engine created' }
{
  data: {
    hash: '2xatxuEuthwpq5e2aQ59JmvuAGagozqtwy7jsTypqWGs',
    instanceHash: 'CTpW7VhRWB5R3xAPT5tUnoSzAwpsrk4e5Hey8fdbusQR'
  },
  level: 'info',
  message: 'runner started'
```

Reply:
and it will be nice to have a loader and the message of the log as status of the loader

Reply:
done, it will now be displayed as outputs of the command eg: 
![](https://github.com/SamVerschueren/listr/raw/master/media/observable.gif)